### PR TITLE
Interrupt Priorities File

### PR DESCRIPTION
**Pull request by MichaelMohn**
# Quality Assurance Checklist 
To make reviews more efficient, please make sure the software feature meets the following standards and check everything off that meets the quality check. Once everything has been checked, the assigned reviewers will begin the review process. _Edit this description to check off the list._

There are exceptions with all guidelines. As long as your decisions are justified, then you are good! Contact the reviewers or the leads about any exceptions. 

## Requirements 
- [x] Followed Coding Style Guide 
- [x] Code Build checks pass 
- [x] No merge conflicts 
- [ ] Software feature has test associated with it. 
- [ ] Software feature has documentation for it updated.
- [x] If applicable, does software feature have simulation test associated with it (.json & out.json file)
- [ ] Test provides useful information and uses relevant data to accurately represent BPS 
    - NOTE: If test file already exists, use that one 
- [ ] 2 Members of the team have already reviewed my PR 
    - NOTE: Approver will not look at PR if 2 other members haven't reviewed first 
- [x] Did you Have Fun?

## Things to Consider 
- [x] Even if the above are checked, is this the best way of writing my code? 
    - It's possible to write code that works, but are there ways to make code more efficient?

### CODE REVIEW

**Comment by MichaelMohn** [comment]
Added the file "Interrupt Priorities" which defines the preempt and sub priority for the UART, CAN, and SPI interrupts.

**Review by connorl309 — DISMISSED** [review]

**Review by connorl309 — COMMENTED** [review]
fix merge conflicts but actual changes look fine

File: BSP/Inc/Interrupt_Priorities.h [hunk 1]
```diff
@@ -0,0 +1,20 @@
+/* Copyright (c) 2018-2022 UT Longhorn Racing Solar */
```
Comment (manthanand) [inline BSP/Inc/Interrupt_Priorities.h:1]: Could you add comments here that follow the coding styles guide?

Comment (MichaelMohn) [reply BSP/Inc/Interrupt_Priorities.h:1]: fixed

**Comment by manthanand** [comment]
> Should this commit include ADC? image

It should include ADC, but shouldn't include things that already have defines for them. There is nothing in ADC that doesn't have defines for it already.

File: BSP/Inc/Interrupt_Priorities.h [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+/* Interrupt_Priorities.h
+ * File to define Preemptive and Sub Interrupt Priorities.
```
Comment (manthanand) [inline BSP/Inc/Interrupt_Priorities.h:3]: This needs to be more descriptive. Imagine the questions you had when you started this project. What layer is this used for? When should this be updated? Any considerations to keep in mind when changing values.

Comment (MichaelMohn) [reply BSP/Inc/Interrupt_Priorities.h]: I'm not sure what to do about "when should this be updated", but I added some more that explained what each one does

**Comment by manthanand** [comment]
This is awesome, thanks

**Pull request merged**